src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginStatus.cshtml

```diff
@@ -3,9 +3,12 @@
 @inject IStringLocalizer<ActiveLogin.Authentication.BankId.AspNetCore.BankIdAuthenticationHandler> Localizer
 
 <div class="card" id="bankIdLoginStatus" style="display: @(Model.AutoLogin ? "block" : "none");">
+
+    <img src="#" title="qr-code" alt="qr-code" id="qr-code" style="display: none;" />
```

Comment:
Remove the title tag, and set alt tag to a localize text. We dont need title, but should use alt for accessibility reasdons.

Swedish: QR-kod för BankID
English: QR Code for BankID

Reply:
of course!